src/components/Trade/TradeConfirm/AssetToAsset.tsx

```diff
@@ -18,19 +19,41 @@ import { TradeAsset } from '../Trade'
 type AssetToAssetProps = {
   sellAsset: TradeAsset
   buyAsset: TradeAsset & Pick<AvatarProps, 'boxSize'>
+  status?: chainAdapters.TxStatus
 } & FlexProps
 
 export const AssetToAsset = ({
   sellAsset,
   buyAsset,
   boxSize = '24px',
+  status,
   ...rest
 }: AssetToAssetProps) => {
-  const sellAssetColor = '#F7931A'
+  const sellAssetColor = !status ? '#F7931A' : '#2775CA'
   const buyAssetColor = '#2775CA'
   const {
     number: { toCrypto, toFiat }
   } = useLocaleFormatter({ fiatType: 'USD' })
+  const gray = useColorModeValue('white', 'gray.750')
+  const red = useColorModeValue('white', 'red.500')
+  const green = useColorModeValue('white', 'green.500')
+
+  const renderIcon = () => {
+    return status === chainAdapters.TxStatus.confirmed ? (
```

Comment:
stylistically i'd prefer a switch here but this is fine

Reply:
This file will be deleted for components/AssetToAsset